### PR TITLE
Add new scenarios for awarding

### PR DESCRIPTION
**Pull request by OrysiaDrabych**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/robot_tests/744)

### CODE REVIEW

File: op_robot_tests/tests_files/awarding.robot [hunk 1]
```diff
@@ -22,20 +23,28 @@ Suite Teardown  Test Suite Teardown
...
-Можливість дочекатись дати початку кваліфікації
+Можливість звірити кількість сформованих авардів лоту
   [Tags]   ${USERS.users['${viewer}'].broker}: Процес кваліфікації
   ...      viewer
   ...      ${USERS.users['${viewer}'].broker}
-  ...      awarding
-  Дочекатись дати початку періоду кваліфікації  ${viewer}  ${TENDER['TENDER_UAID']}
+  ...      number_of_awards
+  [Setup]  Дочекатись закінчення періоду аукціону  ${viewer}  ${TENDER['TENDER_UAID']}
+  Звірити кількість сформованих авардів лоту із ${NUMBER_OF_AWARDS}
```
Comment (ivanka12) [inline op_robot_tests/tests_files/awarding.robot]: для кожного випадку треба буде передавати значення NUMBER_OF_AWARDS?

File: op_robot_tests/tests_files/base_keywords.robot [hunk 2]
```diff
@@ -279,6 +279,11 @@ Resource           resource.robot
   Run as  ${username}  Змінити цінову пропозицію  ${TENDER['TENDER_UAID']}  ${field}  ${value}
 
 
+Можливість зменшити пропозицію до невалідної користувачем ${username}
+  ${value}=  invalid_award_value  ${USERS.users['${tender_owner}'].tender_data.data}
```
Comment (ivanka12) [inline op_robot_tests/tests_files/base_keywords.robot]: чому тут використовується словник tender_owner, а не username, яким будуть виконуватися подальші дії?

File: op_robot_tests/tests_files/base_keywords.robot [hunk 3]
```diff
@@ -335,3 +340,9 @@ Resource           resource.robot
   ...      ${TENDER['TENDER_UAID']}
   ...      ${0}
   Run Keyword And Ignore Error  Remove From Dictionary  ${USERS.users['${viewer}'].tender_data.contracts[0]}  status
+
+
+Звірити кількість сформованих авардів лоту із ${number_of_awards}
+  ${left}=  Convert To Integer  ${number_of_awards}
+  ${right}=  Run As  ${tender_owner}  Отримати кількість авардів в тендері  ${TENDER['TENDER_UAID']}
```
Comment (ivanka12) [inline op_robot_tests/tests_files/base_keywords.robot]: і тут чому tender_owner, чому не передавати username?

**Comment by OrysiaDrabych** [comment]
Review status: 0 of 15 files reviewed at latest revision, 5 unresolved discussions.

---

*[op_robot_tests/tests_files/awarding.robot, line 32 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/744#-L5j6wkeMtYKmMOy9Ud9-r6-32:-L5sXXizWR1r7RWpfQvn:b33kz7) ([raw file](https://github.com/openprocurement/robot_tests/blob/8bde085921f949358e855e11573090bdff4adbaf/op_robot_tests/tests_files/awarding.robot#L32)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

для кожного випадку треба буде передавати значення NUMBER_OF_AWARDS?
</blockquote></details>

та

---

*[op_robot_tests/tests_files/base_keywords.robot, line 283 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/744#-L5j6wkeMtYKmMOy9UdA-r6-283:-L5sX_Utmfh3cUQJqWz3:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/8bde085921f949358e855e11573090bdff4adbaf/op_robot_tests/tests_files/base_keywords.robot#L283)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

чому тут використовується словник tender_owner, а не username, яким будуть виконуватися подальші дії?
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/base_keywords.robot, line 347 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/744#-L5j6wkeMtYKmMOy9UdA-r6-347:-L5sXaMsNFAZicqWK8H5:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/8bde085921f949358e855e11573090bdff4adbaf/op_robot_tests/tests_files/base_keywords.robot#L347)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

і тут чому tender_owner, чому не передавати username?
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/openProcedure.robot, line 725 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/744#-L5j6wkeMtYKmMOy9UdF-r6-725:-L5sXbOWZifYAvPmDJtE:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/8bde085921f949358e855e11573090bdff4adbaf/op_robot_tests/tests_files/openProcedure.robot#L725)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

ТК для provider1,  а теги для provider!
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/openProcedure.robot, line 743 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/744#-L5j6wkeMtYKmMOy9UdF-r6-743:-L5sXbtIyoPrhz-9ulTO:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/8bde085921f949358e855e11573090bdff4adbaf/op_robot_tests/tests_files/openProcedure.robot#L743)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

і тут також відповідно!
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/744)*

File: op_robot_tests/tests_files/initial_data.py [hunk 4]
```diff
@@ -221,6 +221,12 @@ def test_bid_value(max_value_amount, minimalStep):
     })
 
 
+def invalid_award_value(tender_data):
```
Comment (kosaniak) [inline op_robot_tests/tests_files/initial_data.py]: краще поправити цю функцію https://github.com/openprocurement/robot_tests/blob/eauction_cdb2/op_robot_tests/tests_files/initial_data.py#L35 і використати її тут

File: op_robot_tests/tests_files/openProcedure.robot [hunk 5]
```diff
@@ -711,6 +720,15 @@ ${ITEM_MEAT}        ${True}
   Можливість подати цінову пропозицію користувачем ${provider1}
 
 
+Можливість зменшити пропозицію до невалідної для кваліфікації другим учасником
+  [Tags]   ${USERS.users['${provider}'].broker}: Подання пропозиції
```
Comment (kosaniak) [inline op_robot_tests/tests_files/openProcedure.robot]: provider1

File: op_robot_tests/tests_files/openProcedure.robot [hunk 6]
```diff
@@ -711,6 +720,15 @@ ${ITEM_MEAT}        ${True}
...
+  ...      provider1
+  ...      ${USERS.users['${provider}'].broker}
```
Comment (kosaniak) [inline op_robot_tests/tests_files/openProcedure.robot]: provider1

File: op_robot_tests/tests_files/openProcedure.robot [hunk 7]
```diff
@@ -719,6 +737,15 @@ ${ITEM_MEAT}        ${True}
   [Teardown]  Оновити LAST_MODIFICATION_DATE
   Можливість подати цінову пропозицію користувачем ${provider2}
 
+
+Можливість зменшити пропозицію до невалідної для кваліфікації третім учасником
+  [Tags]   ${USERS.users['${provider}'].broker}: Подання пропозиції
```
Comment (kosaniak) [inline op_robot_tests/tests_files/openProcedure.robot]: provider2

File: op_robot_tests/tests_files/openProcedure.robot [hunk 8]
```diff
@@ -719,6 +737,15 @@ ${ITEM_MEAT}        ${True}
...
+  ...      provider2
+  ...      ${USERS.users['${provider}'].broker}
```
Comment (kosaniak) [inline op_robot_tests/tests_files/openProcedure.robot]: provider2

**Comment by kosaniak** [comment]
є конфлікти :(

**Pull request merged**